tests/python/gpu/test_gluon_model_zoo_gpu.py

```diff
@@ -39,7 +39,8 @@ def download_data():
 
 @mx.util.use_np
 @pytest.mark.serial
-@pytest.mark.parametrize('model_name', ['resnet50_v1', 'vgg19_bn', 'alexnet', 'densenet201', 'squeezenet1.0', 'mobilenet0.25'])
+# TODO(vcherepanov): mobilenet0.25 fails this test
+@pytest.mark.parametrize('model_name', ['resnet50_v1', 'vgg19_bn', 'alexnet', 'densenet201', 'squeezenet1.0'])
```

Comment:
I'm OK with removing mobilenet0.25.  I too experienced failures with this test on mobilenet and did not completely troubleshoot it.  I found that it failed non-deterministically on P100 & TU102 when MXNET_GPU_WORKER_NTHREADS=2.